repository/src/main/java/org/apache/atlas/discovery/EntityLineageService.java

```diff
@@ -411,7 +411,7 @@ private void traverseEdges(AtlasVertex datasetVertex, boolean isInput, int depth
                     List<AtlasEdge> qualifyingOutgoingEdges = new ArrayList<>();
                     while (outgoingEdges.hasNext()) {
                         AtlasEdge edge = outgoingEdges.next();
-                        if (edge.getInVertex().equals(lineageContext.getStartDatasetVertex()) || lineageContext.evaluate(edge.getInVertex())) {
+                        if (GraphHelper.getStatus(edge) == AtlasEntity.Status.ACTIVE && (edge.getInVertex().equals(lineageContext.getStartDatasetVertex()) || lineageContext.evaluate(edge.getInVertex()))) {
```

Comment:
Let's use isAllowDeletedProcess flag instead of this, we already this flag at another place

Reply:
Added isAllowDeletedProcess check with ACTIVE edge check
 
_if (lineageContext.isAllowDeletedProcess() || GraphHelper.getStatus(edge) == AtlasEntity.Status.ACTIVE) {_